backend/src/main/java/ai/verta/modeldb/App.java

```diff
@@ -215,7 +215,7 @@ public static void main(String[] args) {
       serverBuilder.intercept(new ExceptionInterceptor());
       serverBuilder.intercept(new MonitoringInterceptor());
       serverBuilder.intercept(new AuthInterceptor());
-      serverBuilder.intercept(new AuditLogInterceptor());
+      serverBuilder.intercept(new AuditLogInterceptor(true));
```

Comment:
We don't want it always true, right? I recall some discussion around that and putting in a config.

Reply:
added config. Also added call count = 1 check

Reply:
added change for this comment https://github.com/VertaAI/verta-uacservice/pull/561#discussion_r577395634